internal/pkg/template/templates/environment/partials/cdn-resources.yml

```diff
@@ -47,7 +50,9 @@ CloudFrontDistribution:
   Type: AWS::CloudFront::Distribution
   Properties:
     DistributionConfig:
-      Aliases: !GetAtt UniqueAliasesAction.UniqueAliases
+      {{- if or .CDNConfig.ViewerCertificate .DelegateDNS}}
+      Aliases: !GetAtt UniqueAliasesAction.UniqueValues
+      {{- end}}
```

Comment:
Several questions come up in my mind 🤔 :
1. Is an empty value like `"[]"` (not `"AWS::NoValue"`) valid for `Aliases` ?
2. Is `Aliases` required if `ViewerCertificate` is used? (I guess it's the other way around: `ViewerCertificate` is required if `Aliases` is used)

Reply:
1. Yes it's valid, it just doesn't fill the resource
2. Yes, iff `ViewerCertificate` does `Aliases` need to be specified (and by definition vise versa)